spec/middleware/authentication_spec.rb

```diff
@@ -39,6 +48,16 @@
         expect(body).to eq([Napa::JsonError.new('bad_password', 'bad password').to_json])
       end
 
+      it 'returns an error message if an incorrect Password header is supplied, and a Passwords header is there' do
```

Comment:
Line is too long. [115/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.